packages/flutter/lib/src/gestures/tap.dart

```diff
@@ -7,13 +7,39 @@ import 'constants.dart';
 import 'events.dart';
 import 'recognizer.dart';
 
-/// Signature for when a pointer that might cause a tap has contacted the screen
-/// at a particular location.
-typedef void GestureTapDownCallback(Point globalPosition);
+/// Details for [GestureTapDownCallback].
```

Comment:
add "such as the position" here so that when you're searching for "position" in a list of classes and their brief descriptions, it comes up.